features/stream/src/main/java/com/m3u/features/stream/components/CustomTextIcon.kt

```diff
@@ -0,0 +1,38 @@
+package com.m3u.features.stream.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun CustomTextIcon(
+    text: String,
+    textColor: Color,
```

Comment:
It should have a default value, Color.Unspecified 
And in the code block, use takeOrElse method to get the LocalContentColor(maybe tv platform) if it's unspecified.

Reply:
example:
val actualContentColor = contentColor.takeOrElse {
if(!tv)
LocalContentColor.current
else 
// import alias
LocalTvContentColor.current
}